### PR TITLE
Add Contributing Template

### LINKED ISSUE #29: Add contributing template
A contrib guideline can look something similar to the one described below:

### Title

That's right, a simple description for issue/pr can go here

### description

Oh! Love the descriptions.

### Solution

Appropriate workaorunds.

### PR DESCRIPTION
**Pull request by rashi-s17**
Resolves #29 

Add CONTRIBUTING.md file which specifies the contribution guidelines to provide an entry point for new contributors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Thank you for your interest in AutoBot. It's an ML Marketplace to buy/sell/auction the datasets of your choice to train ML Models.
+
+## Getting Started
+
+ - Check out AutoBot's [open issues](https://github.com/Abhishek-kumar09/AutoBot/issues) here. Contributions can be made by creating Pull Requests after forking the repository.
+ - Another easy way to start contributing is to *file an issue*. Issues can include bug fixes, adding new features, or updating the documentation that looks outdated.
+ 
+ ### Steps To Contribute:
+ 
+- Fork and clone the repository `git clone https://github.com/Abhishek-kumar09/AutoBot.git`
```
Comment (Abhishek-kumar09) [inline CONTRIBUTING.md:12]: Better is to provide the link of their forked repo.
Like: `git clone https://github.com/<YOUR_USER_NAME>/AutoBot.git`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  `git commit -m "your commit message"`
+- Push the changes `git push origin "branch_name"`
+- Create a PR from your branch 🎉
+
+## New Issue/PR Guidelines:
+
+ ### Title
+ - A simple description of the issue/PR.
+ ### Description
+ - For issues : Describle the feature/bug in detail. In case of bugs, include steps to reproduce.
```
Comment (Abhishek-kumar09) [inline CONTRIBUTING.md:26]: Typo in describe

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Thank you very much for considering Autobot. 
Very minor nitpicks.

Give link to contributing.md in the README

**Comment by rashi-s17** [comment]
I have done the requested changes.

**Comment by Abhishek-kumar09** [comment]
Thanks @rashi-s17 it's wonderful.

**Pull request merged**